examples/convert_little_endian_to_big.cpp

```diff
@@ -30,10 +30,10 @@ int main()
     std::cout << "in is: " << std::hex << in << std::endl;
```

Comment:
Really minor, but when calling the function `is_big_endian` it would make more sense to print `Are we big endian`.